source/features/hide-navigation-hover-highlight.tsx

```diff
@@ -4,14 +4,26 @@ import onetime from 'onetime';
 import features from '../feature-manager.js';
 
 const className = 'rgh-no-navigation-highlight';
+const html = document.documentElement;
 
 function init(): void {
-	document.documentElement.classList.add(className);
-	document.documentElement.addEventListener('navigation:keydown', () => {
-		document.documentElement.classList.remove(className);
+	html.classList.add(className);
+	html.addEventListener('navigation:focus', () => {
```

Comment:
`navigation:keydown` was the same as `keydown`, so the style would get lost even if just `ctrl` was pressed